src/phases/title-phase.ts

```diff
@@ -76,7 +76,7 @@ export class TitlePhase extends Phase {
           this.scene.ui.clearText();
           this.end();
         };
-        if (this.scene.gameData.unlocks[Unlockables.ENDLESS_MODE]) {
+        if (this.scene.gameData.isUnlocked(Unlockables.ENDLESS_MODE)) {
```

Comment:
if you are going to re-use `gameData.[...]` buffer it

```suggestion
        const { gameData } = this.scene;
        if (gameData.isUnlocked(Unlockables.ENDLESS_MODE)) {
```